teacher.rb

```diff
@@ -1,7 +1,7 @@
 require './person'
 
 class Teacher < Person
-  def initialize(age, name = 'Unknown', parent_permission: true)
+  def initialize(specialization = 'Unknown', age = 0, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
```

Comment:
- [ ] The same happens here:

```suggestion
  def initialize(specialization = 'Unknown', age = 0, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
```

🤓 🔧

Reply:
@anagudelogu it is still not working with your solution I'm stuck with this problem in the past 8 hours and no one willing to help me, even most of the reviewer didn't contact me.

Portal is locked now, what I do?